docs/news/1091.bugfix

```diff
@@ -0,0 +1 @@
+Remove some lingering calls to root logger.
```

Comment:
My preference would be to modify the change log for the previous entry which claimed `Log messages no longer go directly to the root logger (#1091)` and move it hear so it is clear where the bug was truly fixed.